Classes/Service/ContentEditableWrapperService.php

```diff
@@ -155,14 +155,15 @@ public function wrapContentWithDropzone(
         $class = 't3-frontend-editing__dropzone';
 
         $dropZone = sprintf(
-            '<div class="%s" ondrop="%s" ondragover="%s" ondragleave="%s" data-new-url="%s" data-moveafter="%d" data-colpos="%d"></div>',
+            '<div class="%s" ondrop="%s" ondragover="%s" ondragleave="%s" data-new-url="%s" data-moveafter="%d" data-colpos="%d" data-defvals=\'%s\'></div>',
```

Comment:
I see but the JSON encoded data inside uses double quotes like:
`{"tx_gridelements_container":279,"tx_gridelements_columns":101}`